package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "open-tourism-platform",
+  "version": "1.0.0",
+  "description": "An open data platform for Nazareth tourism",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/start.js",
+    "lint": "./node_modules/.bin/standard src/*.js",
```

Comment:
Are you sure you need that pattern also: `src/*.js`

My experience with standard is that it just works..